simplemind/settings.py

```
@@ -1,4 +1,4 @@
-from typing import Literal, Optional, Union
+from typing import Literal, Optional, Union, Unpack
```

Ruff

`typing.Unpack` imported but unused

Remove unused import: `typing.Unpack`

(F401)